README.md

```diff
@@ -47,6 +48,7 @@ Micronaut + Camunda = :heart:
 * The process engine can be configured with [generic properties](#generic-properties).
 * The [Camunda REST API and the Webapps](#camunda-rest-api-and-webapps) are supported (currently only for Jetty).
 * The [process engine configuration](#custom-process-engine-configuration) and the [job executor configuration](#custom-jobexecutor-configuration) can be customized programmatically.
+* [Process Engine Plugins](#add-process-engine-plugins) are automatically loaded on start.
```

Comment:
* please move this up one line
* "are automatically loaded on start." -> "are automatically activated on start."